ui/src/AudioRecorder.js

```diff
@@ -0,0 +1,36 @@
+export class AudioRecorder {
+
+    constructor() {
+        this.recordingUrl = "";
+        this.mediaRecorder = "";
```

Comment:
I wouldn't default this.mediaRecorder to a string if it is meant to be some sort of class instance in the future. Setting it to undefined is fine.